doc/help/MERGE.md

```diff
@@ -0,0 +1,44 @@
+kdb-merge(1) -- Three-way merge of KeySets.
+================================
+## DESCRIPTION
+Does a three-way merge between keysets.  
+On success the resulting keyset will be saved to mergepath.  
+On unresolved conflicts nothing will be changed.  
+
+## USAGE
+`kdb merge [options] ourpath theirpath basepath resultpath`  
+
+* ourpath:
+  Path to the keyset to serve as ours  
```

Comment:
Thats kind of a tautology. You should put "ours" in context+perspective. Somehow: From a given base, two different changes occurred. These changed keysets are called ours and theirs...